inputfiles/comments.json

```diff
@@ -1811,11 +1811,11 @@
                 "properties": {
                     "property": {
                         "parent": {
-                            "comment" : "Refers to either the parent WindowProxy, or itself.\n\nIt can rarely be null e.g. for contentWindow of an iframe that is already removed from the parent."
```

Comment:
This was done automatically by prettier, but I can remove it if it is an issue. It does appear to be a valid correction as well.